README.md

```diff
@@ -32,6 +33,19 @@ reactivebleclient.scanForDevices(withServices: [serviceId], scanMode: ScanMode.l
 
 The `withServices` parameter specifies the advertised service IDs to look for. If an empty list is passed, all the advertising devices will be reported. The parameter `scanMode` is only used on Android and follows the conventions described on [ScanSettings](https://developer.android.com/reference/android/bluetooth/le/ScanSettings#SCAN_MODE_BALANCED) Android reference page. If `scanMode` is omitted the balanced scan mode will be used.
 
+
+### Observe host device BLE status
+
+Use the `statusStream` to retrieve updates about the BLE status of the host device (the device running the app) . This stream can be used in order to determine if the BLE is turned on, on the device or if the required permissions are granted. Example usage:
```

Comment:
I believe it's either "Use **the** status stream" or "Use `statusStream`" (no article before a name, "contact Signify", "visit Amsterdam", "use `statusStream`").

Reply:
you are right will fix it thanks!